From ce1dc2af200591b7856336269d7738f53c22ee92 Mon Sep 17 00:00:00 2001
From: Sebastien Boeuf <sebastien.boeuf@intel.com>
Date: Sat, 7 Mar 2015 02:00:14 +0100
Subject: [PATCH 2/2] Fix testCameraToSurfaceTextureMetadata CTS

Enlarging the value of the maximum allowed mismatch in the ov5648
driver ensure us the sensor driver will choose 1280x960 resolution
instead of 2592x1944 if the HAL (and the atomisp driver) are asking
for a CIF or a QCIF resolutions.
In that way, we make sure we will get 30fps resolution selected for
all resolutions during the CTS so as it passes all time.

Issue: android.hardware.cts.CameraGLTest#testCameraToSurfaceTextureMetadata
Change-Id: I57a8ea271c330afe0a0618099aca3aace54cd752
Signed-off-by: Sebastien Boeuf <sebastien.boeuf@intel.com>
---
 drivers/media/i2c/atomisp/ov5648.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/i2c/atomisp/ov5648.c b/drivers/media/i2c/atomisp/ov5648.c
index 01d65ae..4422c42 100644
--- a/drivers/media/i2c/atomisp/ov5648.c
+++ b/drivers/media/i2c/atomisp/ov5648.c
@@ -1387,7 +1387,7 @@ static int ov5648_s_power(struct v4l2_subdev *sd, int on)
  * res->width/height smaller than w/h wouldn't be considered.
  * Returns the value of gap or -1 if fail.
  */
-#define LARGEST_ALLOWED_RATIO_MISMATCH 800
+#define LARGEST_ALLOWED_RATIO_MISMATCH 900
 static int distance(struct ov5648_resolution *res, u32 w, u32 h)
 {
 	unsigned int w_ratio = ((res->width << 13) / w);
-- 
1.9.1

